src/service/viewer-impl.js

```diff
@@ -115,21 +116,15 @@ export class Viewer {
     /** @private {number} */
     this.paddingTop_ = 0;
 
+    /** @private {!Object<string, !Observable<!JSONType>>} */
```

Comment:
We need a special object for these messages. Let's call it a `ViewerMessage` and put this class in this file. It will be similar to ActionInvocation class in https://github.com/ampproject/amphtml/blob/master/src/service/action-impl.js#L69

E.g.
```
export class ViewerMessage {
  constructor(type, data) {
     /** @const {string} */
     this.type = type;
     /** @const {!JSONType} */
     this.data = data;
  }
}
```

Thus, this declaration will become:
```
/** @private {!Object<string, !Observable<!ViewerMessage>>} */
```